tutorials/rise_text.ipynb

```diff
@@ -8,7 +8,7 @@
     "## Interpreting a movie review sentiment model with RISE\n",
```

Comment:
<p>This has nothing to do with this PR, but I'm not happy with this example text for a couple of reasons:</p><ul><li>It is merely positive. As an example we could better have used something with positive and negative elements.</li><li>The short sentence contains a word that both I, and a colleague, didn't know 'delectable'. Not ideal for an example.</li></ul><p>I'll add an issue about this, as it concerns more notebooks.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dianna-ai/dianna/pull/210/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>